hw/top_earlgrey/dv/verilator/verilator_sim_cfg.hjson

```diff
@@ -77,7 +77,7 @@
   ]
 
   // All tests are SW based, so enable this by default.
-  en_run_modes: ["sw_test_mode"]
```

Comment:
Verilator setup and DV setup are completely separate - there is no reuse between the two. THis change will break the Verilator setup (it probably already is though).